helpers.js

```diff
@@ -1,23 +1,52 @@
 const aws = require("aws-sdk");
+const { EKS_SERVICE_URL } = require("./consts");
 const parsers = require("./parsers");
 
-function getEc2(params, settings) {
-  return new aws.EC2({
-    region: parsers.autocomplete(params.region) || settings.region,
-    accessKeyId: params.accessKeyId || settings.accessKeyId,
-    secretAccessKey: params.secretAccessKey || settings.secretAccessKey,
-  });
+const createCorruptedRolePolicyMessage = (roleName) => `Failed to parse role policy for role "${roleName}".`;
```

Comment:
Name looks good, but do we really wrap a template with a function?
It's not reused anywhere, and it doesn't simplify the code, thus there's no benefit (unless there's something I'm missing).

I would just pass the string template directly to the `Error` object constructor.

In future we'll think of creating a common set of reusable errors shared with `utils` (or something similar) library, which will hide such details.